generator/pmdsky_debug_py_generator/loader.py

```diff
@@ -25,7 +25,8 @@
 import yaml
 
 OVERLAY_REGEX = re.compile(r"overlay(\d+)")
-C_TYPE_REGEX = re.compile(r"(((enum )|(struct ))?[a-z0-9_*]+) (([A-Z0-9_]+)(\[\d+])*);")
+C_TYPE_REGEX = re.compile(r"(extern\s+)?(?P<symbol_type>((enum)|(struct)\s+)?[a-z0-9_*]+)\s+"
+                          r"(?P<symbol_name>[A-Z0-9_]+)(?P<array_notation>(\[\d+])*);")
```

Comment:
Under the right circumstances the formatter can break the line in between array dimensions, so we should probably support whitespace between pairs of square brackets. E.g.,

```c
extern uint8_t TEST[256]
                   [256]; // some really long comment that causes the line to break between dims
```

Reply:
Done, thanks!